index.mjs

```diff
@@ -0,0 +1,12 @@
+export * from "./build/Statement.js";
```

Comment:
is it possible to copy this to the `src/` folder? I generally keep all source code in that folder.

Then make sure that it gets copied to `build/` on build.